L1Trigger/L1TTrackMatch/src/L1TkEtMissEmuTrackTransform.cc

```diff
@@ -0,0 +1,123 @@
+#include <cmath>
+#include "L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuTrackTransform.h"
+
+void L1TkEtMissEmuTrackTransform::GenerateLUTs(){
+    phi_quadrants = generate_phi_slice_LUTs (L1TkEtMissEmuAlgo::N_quadrants_);
+    phi_shift     = generate_phi_slice_LUTs (L1TkEtMissEmuAlgo::N_sectors_);
+}
+
+InternalEtWord L1TkEtMissEmuTrackTransform::TransformTrack(TTTrack< Ref_Phase2TrackerDigi_ >& track_ref, float PV){
```

Comment:
Seems to be redundant now that the TTTrack_TrackWord is "finished." Is this on purpose?

Reply:
Fixed in latest commit